### PR TITLE
TiKV configuration: Update description for pin-l0-filter-and-index-blocks

### PR DESCRIPTION
**Pull request by en-jin19**
### What is changed, added or deleted? (Required)

Updated description for pin-l0-filter-and-index-blocks in TiKV configuration document

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/6667
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by en-jin19** [comment]
I have finished self-check of this PR, PTAL, thanks! @TomShawn

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -890,7 +890,7 @@ Configuration items related to `rocksdb.defaultcf`, `rocksdb.writecf`, and `rock
 
 ### `pin-l0-filter-and-index-blocks`
 
-+ Determines whether to pin the index and filter at L0
++ Determines whether to pin the index block and filter block which control the SST file at L0 in memory.
```
Comment (yiwu-arbug) [inline tikv-configuration-file.md:893]:
```suggestion
+ Determines whether to pin the index and filter blocks of the level 0 SST files in memory.
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Review by yiwu-arbug — APPROVED** [review]

**Pull request merged**